src/nccl_ofi_rdma.c

```diff
@@ -2126,6 +2127,15 @@ static int init_send_comm_rails(nccl_net_ofi_rdma_send_comm_t *s_comm,
 		return -EINVAL;
 	}
 
+	/**
+	 * In ENDPOINT_PER_COMM config, the ep address in the handle is not
```

Comment:
Put a TODO comment here.  We shouldn't be doing this, but it's not worth fixing now, given that we're going to do the control channel qp work soonish.